components/Templates/includes/functions-view_template.php

```diff
@@ -116,9 +122,57 @@ function frontier_if_block( $atts, $code ) {
 				break;
 		}
 	} else {
-		$field_data = $pod->field( $atts['field'] );
 
-		$field_type = $pod->fields( $atts['field'], 'type' );
+		/**
+		 * @since 2.7.16 Iterate recursively over magic tag fields (relationships).
+		 */
+		$fields    = explode( '.', $atts['field'] );
+		$field_pod = $pod;
+		$total     = count( $fields );
+		$counter   = 0;
+
+		foreach ( $fields as $field_name ) {
+			$field = $field_name;
+			if ( ++$counter < $total ) {
+
+				if ( 'pick' !== $field_pod->fields( $field, 'type' ) ) {
+					// Relationship type required.
+					break;
+				}
+
+				$entries    = $field_pod->field( $field );
+				$rel_pod    = $field_pod->fields( $field, 'pick_val' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 4 spaces

<sub>hash: b541ce2b30cce36f6d280121fc72ba02</sub>